pkg/plugins/policies/core/rules/merge.go

```diff
@@ -10,57 +10,71 @@ import (
 	jsonpatch "github.com/evanphx/json-patch/v5"
 
 	core_model "github.com/kumahq/kuma/pkg/core/resources/model"
+	"github.com/kumahq/kuma/pkg/util/pointer"
 )
 
 // indicates that all slices that start with this prefix will be appended, not replaced
 const appendSlicesPrefix = "Append"
 
-func MergeConfs(confs []interface{}) (interface{}, error) {
+func MergeConfs(confs []interface{}) ([]interface{}, error) {
```

Comment:
Please add a doc comment for this function, we have to explain `[]interface{}` return value